chart.py

```diff
@@ -1,24 +1,34 @@
 # coding=utf-8
 from dateutil.parser import parse
 
+# How to convert from mg/dL (US format) to mmol/L (non-US format)
+MMOLL_CONVERT_FACTOR = 1.0 / 18.0
```

Comment:
I'm trying to keep the conversion at the "view"/"presentation" layer (thinking of this as a semi-traditional model-view-controller type structure).

It's kinda like storing 'cents' in the DB, and then having code to conver them to "$1.99" in the view code.

I think that the approach OpenAPS has taken is a sensible one (just using one unit of measurement). I'm trying to preserve the sanctity of that, and do the conversion as close to the edge as possible.

EG - if we add code to the system that highlights something in red if sugars are high, the code there won't need a "if unit = mmol/L, then mark_high when greater than X, but if it's unit mg/dL then the high mark is Z"

It looks like I could get the display units with this. It'd require a patch to add the functionality to a report in openaps though.
https://github.com/bewest/decoding-carelink/blob/cff4aea826791244a14727e0e796a1216508718a/decocare/commands.py#L911